index.html

```diff
@@ -0,0 +1,118 @@
+                <h3>Whey Proteins</h3>
+                <img src="img/prd_1672232-MuscleBlaze-Beginners-Protein-1.jpg" width="100%" class="product_img">
+                <p>Whey Protein by MuscleBlaze is the best Indian Whey Protein Powder that helps with quick muscle recovery & prevents muscle breakdown. It’s a complete protein with all the nine essential amino acids. It has a faster and better absorbability as compared to other whey supplements. It is made from premium quality whey isolate protein. You can explore our wide range of best protein powder or protein supplements in India such as Biozyme Whey Protein Isolate, Whey Active, Whey Premium, Whey Prime, etc at affordable prices.</p>  
+            </div>  
+            <div class="product">
+                <h3>Mass Gainer</h3>
+                <img src="img/prot1.jpg" width="100%" class="product_img">
+                <p>MuscleBlaze Super Gainer XXL Chocolate is crafted especially for hardcore fitness enthusiasts who work hard to gain big and sturdy muscles and a sculpted physique.This advanced formula offers 15g protein per 100g serving. It has a blend of fast and slow-releasing proteins.It offers 75g of simple and complex carbs per serving of 100g that ensures a robust and sustained calorie release in your body to fuel your muscles for a longer duration
+                    MuscleBlaze Super Gainer XXL is enriched with 27 vital vitamins and minerals that help in boosting the immunity and regulating enzymes in your body</p>
+                    
+            </div>
+            <div class="product">
+                <h3>Biozyme Performance Whey</h3>
+                <img src="img/pro3.jpg" width="100%" class="product_img">
+                <p>Whey this is the best proteinProtein by MuscleBlaze is the best Indian Whey Protein Powder that helps with quick muscle recovery & prevents muscle breakdown. It’s a complete protein with all the nine essential amino acids. It has a faster and better absorbability as compared to other whey supplements. It is made from premium quality whey isolate protein. You can explore our wide range of best protein powder or protein supplements in India such as Biozyme Whey Protein Isolate, Whey Active, Whey Premium, Whey Prime, etc at affordable prices.</p>
+                
+            </div>
+        </div>
+    </section>
+
+    
+    <div class="btn-group">
+        
+        <a class="btn btn-accent">See all our products</a>
+        
+    </div>
+         <h3 class="order_intro">Or Order through</h3>
+    <div class="btn-group">
+        <a href="" class="btn btn-primary">Instagram</a>
+        <a href="" class="btn btn-primary">Whatsapp</a>
+        <a href="" class="btn btn-primary">Facebook</a>
+    </div>
+
+    <footer>
+       <ul class="footer-nav">
+            <li><a href="">Home</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Product</a></li>
+            <li><a href="">Connect</a></li>            
+       </ul>
```

Comment:
Can we use `nav` here instead of using `ul` and adding nav classes

Reply:
lemme try this one..